src/anyon/qpu_interface.jl

```diff
@@ -4,12 +4,49 @@ using HTTP
 using JSON
 using TOML
 
+"""
+    Status
+
+A data structure that stores the status of a quantum computation.  
+# Fields
+- `type::String` -- Word describing the status of the computation.
```

Comment:
`type::String` -- One of the declared types, e.g.:     
    -"QUEUED"   : Computation in queue
    -"RUNNING"  : Computation being processed
    -"FAILED"   : QPU service has returned an error message
    -"SUCCEEDED": Computation is completed, result is available.
    -"CANCELLED": Computation was terminated before being executed.